### ISSUE TITLE
Discrepancies with the web app/ weird emotions value

### ISSUE THREAD

**Issue by michelesergiowork** [aman2656/text2emotion-library#7]
I first tested text2emotions with the online app, then I tried to use it in my Jupiter notebook. The results differ. 

For instance the web app assigns no emotional value to "make", but the python package classifies it as "sad". To me, it doesn't make any sense. Similarly terms as "hope" and "horny" display different emotional values online and in jupyter. In the emotion dictionary used in the get_emotion function there are werid keys like "Rockfeller Center". The ones online are far more correct!

What version of the package is used by the online app? How can I reproduce those results? Are you sure the dictionary that defines emotional values is correct? Maybe something went wrong along the way! It is a great package but it is hard to use it like this!

**Comment by michelesergiowork** [comment]
I found the code of the app, the emotion dictionary used in this repository is still different from the one used by the app! https://github.com/Amey23/pyemotion-demo/blob/master/functions.py

**Comment by Hunter2809** [comment]
Hey @michelesergiowork! 

Have u found any breakthrough in this issue? Since this will be damn helpful to everyone.

**Comment by michelesergiowork** [comment]
No, my advice is not to use this package. They attribute emotional values to words that should not have one. Just look in the code! There are other options like IBM tone analyzer (not-free) and deepmoji. But I haven't found anything that works well in the end!

**Comment by Hunter2809** [comment]
Well, yea I agree that this module isn't that great. But then, (in my honest opinion), we don't get to complain since the module is one of it's kind since I never really saw anything like this before and it's free too. But the only downside is, where nltk and keras take only 7 seconds to process a text, it takes more than 5 min (and still processing) for this module to do the same but nothing!

**Comment by michelesergiowork** [comment]
It did not take that long for me. Just take a second look at the emotional dictionaries used by the package before proceeding with it! I agree that the model has got some great features and it is one of a kind, but the dictionaries are not reliable!

https://github.com/aman2656/text2emotion-library/blob/master/Text2Emotion/__init__.py
